spec/test_apps/app/snapshots/books/create/0.json

```diff
@@ -1,7 +1,20 @@
 {
   "request": {
     "method": "POST",
-    "url": "http://www.example.com/books"
+    "url": "http://www.example.com/books",
```

Comment:
Method and URL are now duplicated in env/headers. We could flatten `request` to be just the env.

Reply:
@jdufresne  I lean towards leaving it for the sake of API update minimalism and the focus of my time in this system and our other system, but am open to doing this if you suggest it even with those things in mind. Though, I would opt to only remove the `request.method` entry from top level as `request.url` does some work in the background that seems worth keeping on top of the raw header data.